modules/standard/Path.chpl

```diff
@@ -610,6 +624,95 @@ proc file.realPath(out error: syserr): string {
   return "";
 }
 
+// Compute the common prefix length between two lists of path components.
+private proc commonPrefixLength(a1: [] string, a2: [] string): int {
+  // Can we use the builtin min/max functions for this?
```

Comment:
Yes, but what you have is better than calling `min()` and `max()` - I'd drop this comment.